backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts

```diff
@@ -59,9 +59,16 @@ export class TableOrdersController {
   @ApiUnprocessableEntityResponse({ type: TableOrderAlreadyBilledException, description: 'TableOrder is already billed' })
   @Post(':tableOrderId')
   async addMenuItemToTableOrder(@Param() getTableOrderParams: GetTableOrderParams, @Body() addMenuItemDto: AddMenuItemDto): Promise<TableOrder> {
+    console.log('passing by here iam a controller');
```

Comment:
Can you remove or change this log for something a bit more useful? 😓

Reply:
debugging